### PR TITLE
#566 - Fix proselint bugs

### PR DESCRIPTION
**Pull request by Timherlaud**
#566 - Fix proselint bugs

### CODE REVIEW

File: core/file-upload.md [hunk 1]
```diff
@@ -252,5 +252,5 @@ uploaded cover, you can have a nice illustrated book record!
 }
 ```
 
-Voila! You can now send files to your API, and link them to any other resources
+You can now send files to your API, and link them to any other resources
```
Comment (alanpoulain) [inline core/file-upload.md]: I think using `Voilà` will solve the issue.

File: core/serialization.md [hunk 2]
```diff
@@ -403,8 +403,8 @@ final class BookContextBuilder implements SerializerContextBuilderInterface
     public function createFromRequest(Request $request, bool $normalization, ?array $extractedAttributes = null): array
     {
         $context = $this->decorated->createFromRequest($request, $normalization, $extractedAttributes);
-        $resourceClass = $context['resource_class'] ?? null;
-        
+        $resourceClass = isset($context['resource_class']) ? $context['resource_class'] : null;
```
Comment (alanpoulain) [inline core/serialization.md]: You shouldn't change this. Maybe there's a way for proselint to understand it's code.

File: deployment/kubernetes.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 Both [Google Cloud Platform](https://cloud.google.com/kubernetes-engine/), [Microsoft Azure](https://azure.microsoft.com/en-us/services/container-service/kubernetes/)
 and [Amazon Web Services](https://aws.amazon.com/eks/) provide managed Kubernetes environment.
 
-[The official API Platform distribution](../distribution/index.md) contains a built-in [Helm](https://helm.sh/) (the k8s
+[The official API Platform distribution](../distribution/index.md) contains a builtin [Helm](https://helm.sh/) (the k8s
```
Comment (alanpoulain) [inline deployment/kubernetes.md]: It was asked by proselint?

File: core/file-upload.md (hunk 1 above)
Comment (Tim59) [reply core/file-upload.md]: I will try thanks

File: core/serialization.md (hunk 2 above)
Comment (Tim59) [reply core/serialization.md]: ok

File: deployment/kubernetes.md (hunk 3 above)
Comment (Tim59) [reply deployment/kubernetes.md]: Yes for me it was this error : garner.phrasal_adjectives.ly No hyphen is necessary in phrasal adjectives with an adverb ending in -ly, unless the -ly adverb is part of a longer phrase

Comment (dunglas) [reply deployment/kubernetes.md]: But it doesn't end by `-ly`. `built-in` looks like the recommended form, so please keep it: https://ell.stackexchange.com/a/16887

File: core/file-upload.md (hunk 1 above)
Comment (Tim59) [reply core/file-upload.md]: Yes it work thanks

File: core/serialization.md (hunk 2 above)
Comment (Tim59) [reply core/serialization.md]: it's seem to notify each double "?" 
["[a-z]*\?{2,}"](https://github.com/amperser/proselint/blob/f80783f47367d1d9c08e5866afe0af39a05d4a7e/proselint/checks/hyperbole/misc.py)

**Comment by alanpoulain** [comment]
I think the `ly` issue is because of `fully-featured`.

**Comment by alanpoulain** [comment]
Maybe we can remove the rule `hyperbolic.misc` for writing PHP. WDYT @dunglas?

**Comment by Tim59** [comment]
@alanpoulain do you see other change needed ? thx

**Comment by alanpoulain** [comment]
No but I'm waiting for @dunglas's opinion :slightly_smiling_face:

**Pull request closed without merging**